src/plugins/channelTabs/components/BookmarkContainer.tsx

```diff
@@ -69,6 +69,24 @@ function Bookmark({ bookmarks, index, methods }: { bookmarks: Bookmarks, index:
     const bookmark = bookmarks[index];
 
     const ref = useRef<HTMLDivElement>(null);
+
+    let unreadCount, mentionCount;
+    if (!("bookmarks" in bookmark)) { // Not a folder
+        const channel = ChannelStore.getChannel(bookmark.channelId);
+
+        [unreadCount, mentionCount] = useStateFromStores(
```

Comment:
this conditionally calls a hook. you'd need to either move `useStateFromStores` out of the if statement and always return 0 for folders, or move this logic to `NotificationDot`